core/processor/src/main/java/io/quarkus/annotation/processor/util/FilerUtil.java

```diff
@@ -56,6 +61,25 @@ public void write(String filePath, String value) {
         }
     }
 
+    public List<String> read(String filePath) {
```

Comment:
I fear using `read` is going to be annoying if at some point, we want to have a method to read a string.

We should also rename `void write(String filePath, Set<String> set)` to `writeSet`. And `write(String filePath, Properties properties)` to `writeProperties`.

```suggestion
    public Set<String> readSet(String filePath) {
```